upgrade-2.x.sh

```diff
@@ -0,0 +1,201 @@
+#!/bin/bash
```

Comment:
I've never seen a `.bash` file. Out of all `.sh` files in my system's `/usr`, 40% are bash scripts